examples/src/lib.rs

```diff
@@ -229,17 +230,30 @@ impl ExampleBase {
                 .to_vec();
             extension_names.push(DebugUtils::name().as_ptr());
 
+            #[cfg(any(target_os = "macos", target_os = "ios"))]
+            {
+                extension_names.push(KhrGetPhysicalDeviceProperties2Fn::name().as_ptr());
+                extension_names.push(KhrPortabilityEnumerationFn::name().as_ptr());
+            }
+
             let appinfo = vk::ApplicationInfo::default()
                 .application_name(app_name)
                 .application_version(0)
                 .engine_name(app_name)
                 .engine_version(0)
                 .api_version(vk::make_api_version(0, 1, 0, 0));
 
+            #[cfg(any(target_os = "macos", target_os = "ios"))]
+            let create_flags = vk::InstanceCreateFlags::ENUMERATE_PORTABILITY_KHR;
+
+            #[cfg(not(any(target_os = "macos", target_os = "ios")))]
+            let create_flags = vk::InstanceCreateFlags::default();
```

Comment:
I usually prefer _not_ having the `#[cfg()]` clause duplicated (can be done with `cfg_if`), and compile-test both "blocks" if they're not compiletime-dependent on the given cfg:

```suggestion
            let create_flags = if cfg(any(target_os = "macos", target_os = "ios")) {
                vk::InstanceCreateFlags::ENUMERATE_PORTABILITY_KHR
            else
                vk::InstanceCreateFlags::default()
            };
```

Reply:
Cool, I didn't know about that, thanks!